pycbc/psd/analytical_space.py

```diff
@@ -58,8 +62,78 @@ def psd_lisa_acc_noise(f, acc_noise_level=3e-15):
     return s_acc_nu
 
 
-def psd_lisa_oms_noise(f, oms_noise_level=15e-12):
-    """ The PSD of LISA's OMS noise.
+def psd_lisa_acc_noise(f, acc_noise_level=3e-15):
+    """ The PSD of LISA's acceleration noise.
+
+    Parameters
+    ----------
+    f : float or numpy.array
+        The frequency or frequency range, in the unit of "Hz".
+    acc_noise_level : float
+        The level of acceleration noise.
+
+    Returns
+    -------
+    s_acc_nu : float or numpy.array
+        The PSD value or array for acceleration noise.
+    Notes
+    -----
+        Please see Eq.(11-13) in <LISA-LCST-SGS-TN-001> for more details.
+    """
+    s_acc_nu = _psd_acc_noise(f, acc_noise_level)
+
+    return s_acc_nu
+
+
+def psd_tianqin_acc_noise(f, acc_noise_level=1e-15):
+    """ The PSD of TianQin's acceleration noise.
+
+    Parameters
+    ----------
+    f : float or numpy.array
+        The frequency or frequency range, in the unit of "Hz".
+    acc_noise_level : float
+        The level of acceleration noise.
+
+    Returns
+    -------
+    s_acc_nu : float or numpy.array
+        The PSD value or array for acceleration noise.
+    Notes
+    -----
+        Please see Table(1) in <10.1088/0264-9381/33/3/035010>
+        for more details.
+    """
+    s_acc_nu = _psd_acc_noise(f, acc_noise_level)
```

Comment:
In Eq. (120), why `S_acc_nv` is related to `S_x`? Is this a typo? In the LISA case, it's `S_a`.

Reply:
I also need to update TianQin's sensitivity curve form.

Reply:
Thanks, that is a typo, we will update it in the next version!

Reply:
@HumphreyWang  I have a question here, do they share the same sensitivity equation (this is from LDC manual):
![image](https://github.com/gwastro/pycbc/assets/6574526/8a59de74-f105-4ddb-84f3-b5a6b1f6b686)
Just different in the PSD of acc/oms noise, arm length, and <Fp+>?

Reply:
@WuShichao Right, although they have different acc/oms noise, the convention between PSD and sensitivity follows the same.

Reply:
@HumphreyWang, the current code has addressed all your questions. However, I find an inconsistency between the implementation of this PR and the previous TianQin paper. Let me show you:
![image](https://github.com/gwastro/pycbc/assets/6574526/c7be6e14-a62d-4b67-b9e6-927890c9950f)
I use this simplified equation for the dashed black curve:
![image](https://github.com/gwastro/pycbc/assets/6574526/15110f85-d03c-4d7b-a7a2-646dee51b291)
Do you know the reason why my plot is higher than Fig. 2 in http://arxiv.org/abs/2005.07889, but consitent with Fig.3 in https://link.aps.org/doi/10.1103/PhysRevD.100.043003 ?

Reply:
@WuShichao The black one, as the equation you showed here, removed the sky-averaged factor 10/3. You can check Eq. (11) in 2005.07889 and the following descriptions for explanation.

So as far as I understand, for consistency, the confusion noise described in that paper should directly be added into non-averaged sensitivity, and then multiply the factor. (I'll double-check this later, thanks for the question!)

Reply:
@HumphreyWang You can see Eq.(15) in https://link.aps.org/doi/10.1103/PhysRevD.100.043003. It doesn't remove 3/10. I prefer to keep it so that we can be consistent with the LDC convention, I'm sure the LISA one is correct, and TianQin cannot be below LISA's curve.

Reply:
@HumphreyWang I can reproduce the confusion plot:
![image](https://github.com/gwastro/pycbc/assets/6574526/40d7829c-df0e-4cc1-9ec8-ef7cccc6f69f)

Reply:
@WuShichao Whether we should use/remove the factor of 10/3 depends on the situation, for here I agree to keep it. However, the confusion noise for TianQin is based on the non-averaged one, so it should also be multiplied by the factor.